includes/Api/ApiLicenseValidation.php

```diff
@@ -75,6 +75,7 @@ public function register_reset_invalid_license_route() {
 					}
 					wp_cache_delete( 'wubtitle_token', 'options' );
 					wp_cache_delete( 'wubtitle_token_time', 'options' );
+					$token            = isset( $headers['token'][0] ) ? $headers['token'][0] : '';
```

Comment:
```suggestion
					$token            = $headers['token'][0] ?? '';
```
prova questa sintassi